src/test/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/controllers/OcrValidationControllerTest.java

```diff
@@ -167,6 +167,21 @@ void should_return_200_with_form_not_found_error_when_form_type_case_does_not_ma
                 .json("{\"warnings\":[],\"errors\":[\"Form type 'Personal' not found\"],\"status\":\"ERRORS\"}"));
     }
 
+    @Test
+    void should_return_403_status_when_both_s2s_token_and_form_type_are_invalid() throws Exception {
```

Comment:
this test doesn't validate the invalid form type as auth service will throw `ForbiddenException` straight away

Reply:
I guess the name is not clear.
What I had in mind here was to check that when s2s token is invalid we just return 403 (previously it would first check the type, and return 200 with error about the form type, even when s2s token is invalid,

Open to suggestion on better test name.